Phylo/switch to ppx

rneher opened the pull request.

joverlee521 commented:
Thanks for flagging for review! I pushed up some fixes (https://github.com/nextstrain/measles/compare/8a545705bfc5548c41a660f98c91cbcb5e3ea7fe...da20b6cd025d4110e24c1f349303e80468009067) for issues that I considered blockers.

Other outstanding tasks that we should follow up on but aren't necessarily blockers for this PR:
- [ ] update ingest/annotations.tsv to PPX accessions
- [ ] update phylo/include + dropped strains to PPX accessions
- [ ] update phylo/example_data
- [ ] determine if we want to restore the `is_reference` column, which is used for the [WHO reference color-by in N450](https://nextstrain.org/measles/N450@2025-12-01?c=is_reference)
- [ ] remove NCBI configs/scripts/example_data from ingest

rneher commented:
I translated the include/exclude files and the annotation. For the `is_reference` to work again, we first need an empty `is_reference` column in the metadata? like [here](https://github.com/nextstrain/measles/blob/main/ingest/rules/fetch_from_ncbi.smk#L93C15-L93C57)?

rneher commented:
@joverlee521 I addressed most issues. I am not sure whether we want to remove the NCBI stuff. It is true that the changes to curate mean that one can't easily switch back... but having some NCBI ingest code around is also not that bad.
